tests/pytests/test_index_error.py

```diff
@@ -240,6 +293,50 @@ def test_partial_doc_index_failures(env):
     env.assertEqual(info['field statistics'][0], expected_text_stats)
     env.assertEqual(info['field statistics'][1], excepted_numeric_stats)
 
+def test_multiple_index_failures(env):
+    # Create 2 indices with a different schema order.
+    env.expect('ft.create', 'idx1', 'SCHEMA', 'n1', 'numeric', 'n2', 'numeric').ok()
+    env.expect('ft.create', 'idx2', 'SCHEMA', 'n2', 'numeric', 'n1', 'numeric').ok()
+
+    # Create a document with two fields containing invalid numeric values.
+    env.expect('HSET', 'doc', 'n1', 'banana', 'n2', 'meow').equal(2)
+
+    index_to_errors_strings = {'idx1': 'banana', 'idx2': 'meow'}
+
+    for idx in ['idx1', 'idx2']:
+        info = index_info(env, idx)
+
+        expected_error_dict = {
+            indexing_failures_str: 1,
+            last_indexing_error_str: f"Invalid numeric value: '{index_to_errors_strings[idx]}'",
+            last_indexing_error_key_str: 'doc'
+        }
+
+        # Both indices contain one error for the same document.
+        error_dict = to_dict(info["Index Errors"])
+        env.assertEqual(error_dict, expected_error_dict)
+
+
+        # Each index failed to index the doc due to the first failing field in the schema.
+        index_to_failed_field = {'idx1': 'n1', 'idx2': 'n2'}
+        index_to_ok_field = {'idx1': 'n2', 'idx2': 'n1'}
+        expected_failed_field_stats = [
+            'identifier', index_to_failed_field[idx], 'attribute', index_to_failed_field[idx], 'Index Errors',
+            ['indexing failures', 1, 'last indexing error',
+             f"Invalid numeric value: '{index_to_errors_strings[idx]}'",
+             'last indexing error key', 'doc']
+        ]
+
+        expected_no_error_field_stats = [
+            'identifier', index_to_ok_field[idx], 'attribute', index_to_ok_field[idx], 'Index Errors',
+            ['indexing failures', 0, 'last indexing error', 'N/A', 'last indexing error key', 'N/A']
+        ]
+
+        env.assertEqual(info['num_docs'], 0)
+        env.assertEqual(info['field statistics'][0], expected_failed_field_stats)
+        env.assertEqual(info['field statistics'][1], expected_no_error_field_stats)
```

Comment:
I understand why that is correct, but looking at it now, it might be misleading... Perhaps it would be better if we at least increase the `indexing failures` on each field in the spec anyway once we encounter an error in one of the fields, so it won't look like a partial indexing was made. WDYT?

Reply:
I think it makes the per-field indexing error counter redundant, as a we would like to identify field-types with higher errors amount

Reply:
i don't see a reason why it may look like partial indexing since the error is counted in the index level as well.

Reply:
I see what you mean. How about trying to index each field instead of failing fast and setting the error counter for the bad fields? It would increase latency but in error code path we should care less. How big of a change is that?